crates/sargon/src/system/sargon_os/sargon_os_accounts.rs

```diff
@@ -778,7 +792,7 @@ impl SargonOS {
                 Ok(())
             }
             .and_then(|_| {
-                p.assert_factor_instances_valid().inspect_err(|_| { p.networks = networks_backup; })
+                p.assert_new_factor_instances_not_already_used(instances_of_new_entities.clone()).inspect_err(|_| { p.networks = networks_backup; })
```

Comment:
We used to assert that FactorInstances are not shared between entities against the WHOLE profile, now we assert the NEW against the current only.